src/SwedbankPay.Sdk.Infrastructure/OperationListDto.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace SwedbankPay.Sdk
+{
+    public class OperationListDto : List<HttpOperationDto>
+    {
+        public OperationListDto()
+        {
+        }
+
+        public OperationListDto(IEnumerable<HttpOperationDto> operations)
+            : base(operations)
+        {
+        }
+
+        public override string ToString()
+        {
+            return string.Join(", ", this.Select(o => o.Rel));
```

Comment:
@asbjornu if I remove the `this.` here I will get a compilation error.
`The name Select does not exist in the current context`

Reply:
But it's the only `this.` left in the solution, I hope you can forgive this transgression.